pom.xml

```diff
@@ -379,6 +379,12 @@
                 <version>${jackson.version}</version>
             </dependency>
 
+            <dependency>
```

Comment:
Since this and the others are (to me anyway) not really obvious why they're here, would it be good to add some comments explaining why each one is here?

Reply:
We'll add the comments later..